### PR TITLE
Adjust Copy For EmptyCardGrid When TopToolbar

### PR DESCRIPTION
**Pull request by evancrow**
Changes the copy for EmptyCardGrid to say above instead of below when the top toolbar is active. Also, probably need a similar UI for spaces.

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

File: Client/Frontend/Browser/Card/EmptyCardGrid.swift [hunk 1]
```diff
@@ -6,13 +6,14 @@ import SwiftUI
 
 struct EmptyCardGrid: View {
     let isIncognito: Bool
+    var isTopBar: Bool = false
```
Comment (EdwardJTL) [inline Client/Frontend/Browser/Card/EmptyCardGrid.swift:9]: can this be a let?

**Review by EdwardJTL — COMMENTED** [review]
could you add some screenshots for inline vs not inline after the change?

File: Client/Frontend/Browser/Card/EmptyCardGrid.swift (hunk 1 above)
Comment (evancrow) [reply Client/Frontend/Browser/Card/EmptyCardGrid.swift:9]: Well that depends... do we want false to be the default value or do we not want one?

**Comment by evancrow** [comment]
Here's the screenshot (the incognito version has the top toolbar copy):

Screen Shot 2022-04-19 at 9 11 24 PM

**Review by EdwardJTL — APPROVED** [review]

**Pull request merged**